dbt_style_guide.md

```diff
@@ -103,19 +164,71 @@ select * from filtered_events
 ## SQL style guide

```

Comment:
can we change this CTE name to something other than `filtered_events`? confusing name since we're trying to convey that filters should not happen in this CTE

Reply:
👍

Reply:
I missed this one - changed! [Commit](https://github.com/dbt-labs/corp/pull/63/commits/bda4126b5b0b71be6aeb3ab0fa749d008beb8b70)